scripts/build.js

```diff
@@ -26,20 +60,14 @@ try {
   return;
 }
 
-const oldSchema = buildSchema(getMasterSchema());
-const newSchema = buildSchema(schema);
-const breakages = findBreakingChanges(oldSchema, newSchema);
-
-if (breakages.length === 0) {
-  console.log(chalk.green("Changes are backwards compatible"));
-} else {
-  console.error(chalk.red("Breaking changes found"));
-  console.error("Please deprecate these fields instead of removing:");
-
-  breakages.forEach(breakage => {
-    console.error(`  ${breakage.type}: ${breakage.description}`);
+getMasterSchema()
+  .then(oldSchema => {
+    findBreakingChangesBetweenCurrentAndPrevious(oldSchema, schema);
+  })
+  .then(() => {
+    deleteClonedRepo();
+  })
+  .catch(error => {
+    console.error(chalk.red(error));
+    deleteClonedRepo();
```

Comment:
Rather than call `deleteClonedRepo()` in two places, I think you can use a [finally](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally). e.g.

```js
getMasterSchema()
.then(oldSchema => findBreakingChanges(oldSchema, newSchema))
.catch(error => console.error(chalk.red(error)))
.finally(() => deleteClonedRepo());
```